website/pages/blog/aws-priv-esc-identity-center.mdx

```diff
@@ -0,0 +1,266 @@
+
+## Introduction to AWS Identity Center
+
+AWS Identity Center [has requirements for](https://docs.aws.amazon.com/singlesignon/latest/userguide/get-started-prereqs-considerations.html) use including using AWS Organizations for managing AWS Accounts.
+
+Identity Center must be run from either the Organizational management account or a delegated administrator account.  For more information about delegated administrator accounts, see [our blog post and research on delegated administrator](https://www.cloudquery.io/blog/guide-aws-org-delegation).
+
+Since usage of AWS Identity Center requires usage of AWS Organizations and can only be managed from either the organizational management account or delegated administrator account, privilege escalation with AWS Identity Center requires access to whichever account is used for management of AWS Identity Center: either the organizational management account or the delegated administrator account used for AWS Identity Center.
+
+Note: it is still possible to escalate privilege in member accounts without AWS Identity Center using direct modification of the IAM Roles and Policies tied to AWS Identity Center such as `iam:AttachRolePolicy`.
+
+An example Role Trust Policy for an Identity Center created IAM Role `arn:aws:iam::123412341234:role/aws-reserved/sso.amazonaws.com/eu-central-1/AWSReservedSSO_AWSReadOnlyAccess_1111111111111111` in a target account looks as follows.  
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Federated": "arn:aws:iam::123412341234:saml-provider/AWSSSO_1111111111111111_DO_NOT_DELETE"
+            },
+            "Action": [
+                "sts:AssumeRoleWithSAML",
+                "sts:TagSession"
+            ],
+            "Condition": {
+                "StringEquals": {
+                    "SAML:aud": "https://signin.aws.amazon.com/saml"
+                }
+            }
+        }
+    ]
+}
+```
+
+### Dual Authorization
+
+For IAM Identity Center, certain APIs support [dual authorization](https://docs.aws.amazon.com/singlesignon/latest/userguide/security-authorization.html).  AWS released newer API operations after October 15th, 2020.  For any IAM Identity Center instances created before October 15th, 2020, both API actions can be used.  Instances created after October 15th, 2020 use the [newer API actions](https://docs.aws.amazon.com/singlesignon/latest/APIReference/API_Operations.html).  
+
+For example, to add an inline policy, both `PutPermissionPolicy` and `PutInlinePolicyToPermissionSet` could work.
+
+![Dual Authorization Table Provided by AWS](/images/blog/aws-priv-esc-identity-center/dualauth.png)
+
+Dual Authorization Table Provided by AWS
+
+## Privilege Escalation with AWS Identity Center (SSO)
+
+Identity Center has multiple service prefixes including:
```

Comment:
I think I miss a piece here or a clarification is needed. are those permissions that if they are granted the user can add himself permissions and/or himself to groups? If yes, better to state that then explicitly.

Reply:
Added!